source/gameengine/Rasterizer/RAS_2DFilterData.h

```diff
@@ -43,6 +43,8 @@ class RAS_2DFilterData
 	std::vector<std::string> propertyNames;
 	/// The KX_GameObject (or something else?) that provides the values for the uniforms named above.
 	CValue *gameObject;
+	/// Enable/Disable mipmap in 2DFilter actuator at conversion
+	int mipmap;
```

Comment:
There is everywhere int before conversion in ras-2dfilter. Due to bool not supported in dna